scripts/foundry/transmuter/TransmuterAddCollateralMWEURC.s.sol

```diff
@@ -0,0 +1,156 @@
+    uint64[] public xFeeMint;
+    int64[] public yFeeMint;
+    uint64[] public xFeeBurn;
+    int64[] public yFeeBurn;
+    uint256 public capStablecoin;
+    address public agToken;
+
+    function run() external {
+        uint256 chainId = vm.envUint("CHAIN_ID");
+
+        ITransmuter transmuter = ITransmuter(_chainToContract(chainId, ContractType.TransmuterAgEUR));
+        agToken = address(transmuter.agToken());
+        bytes memory transactions;
+        uint8 isDelegateCall = 0;
+        address to = address(transmuter);
+        uint256 value = 0;
+
+        xFeeBurn = new uint64[](3);
+        xFeeMint = new uint64[](3);
+        yFeeMint = new int64[](xFeeMint.length);
+        yFeeBurn = new int64[](xFeeBurn.length);
+        xFeeBurn[0] = 1e9;
+        xFeeBurn[1] = 0.21e9;
+        xFeeBurn[2] = 0.20e9;
+        yFeeBurn[0] = 0.005e9;
+        yFeeBurn[1] = 0.005e9;
+        yFeeBurn[2] = int64(uint64(MAX_BURN_FEE));
+
+        xFeeMint[0] = 0;
+        xFeeMint[1] = 0.59e9;
+        xFeeMint[2] = 0.60e9;
+        yFeeMint[0] = 0.0005e9;
+        yFeeMint[1] = 0.0005e9;
+        yFeeMint[2] = int64(uint64(MAX_MINT_FEE));
+
+        capStablecoin = 2_000_000 ether;
+
+        // Add the new collateral
+        {
+            {
+                bytes memory data = abi.encodeWithSelector(ISettersGovernor.addCollateral.selector, COLLATERAL_TO_ADD);
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+            {
+                // Mint fees
+                bytes memory data = abi.encodeWithSelector(
+                    ISettersGuardian.setFees.selector,
+                    COLLATERAL_TO_ADD,
+                    xFeeMint,
+                    yFeeMint,
+                    true
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+            {
+                // Burn fees
+                bytes memory data = abi.encodeWithSelector(
+                    ISettersGuardian.setFees.selector,
+                    COLLATERAL_TO_ADD,
+                    xFeeBurn,
+                    yFeeBurn,
+                    false
+                );
+                uint256 dataLength = data.length;
+                bytes memory internalTx = abi.encodePacked(isDelegateCall, to, value, dataLength, data);
+                transactions = abi.encodePacked(transactions, internalTx);
+            }
+
+            {
+                address oracle = 0xa7ea0d40C246b876F76713Ba9a9A95f3f18AB794;
+                uint256 normalizationFactor = 1e18;
+                bytes memory targetData = abi.encode(1008235463728948111);
```

Comment:
are we sure of this target data? Seems like the current target price is invalid -> should be 1002...

Reply:
Yes this is because I took the value when doin the tests
And this value is higher than direct `previewMint` on the vault, because we compose with EURC/USD and USD/EUR oracle so it increases a bit the value because of approximation